README.md

```diff
@@ -1398,6 +1400,53 @@ You can use these placeholders to create custom queries to filter and migrate a
 
 > When creating custom queries via this job, please remember to confirm that B2C Customer Profiles exist that satisfy your constructed query.  Any query that returns zero results will generate a NO DATA message in the job log-files.
 
+### Data mapping between B2C Commerce and the Core Platform
+
+As the B2C Commerce customer profile and its addresses are fetched by the core platform, the core platform holds the data-mapping between the core attributes and the B2C ones. You can find hereafter the detail of this mapping. The `Core Alt ID` is used in place of the `Core ID` when person accounts is enabled. The `Core ID` is used when person accounts are not enabled:
+
+| Label               | Core Object            | Core ID                    | Core Alt ID                 | B2C Object      | OCAPI ID            |
+|:-------------------:|:----------------------:|:--------------------------:|:---------------------------:|:---------------:|:-------------------:|
+| Address Id          | B2C_CustomerAddress__x | AddressId__c               | (None)                      | CustomerAddress | address_id          |
+| Address 1           | B2C_CustomerAddress__x | Address1__c                | (None)                      | CustomerAddress | address1            |
+| Address 2           | B2C_CustomerAddress__x | Address2__c                | (None)                      | CustomerAddress | address2            |
+| City                | B2C_CustomerAddress__x | City__c                    | (None)                      | CustomerAddress | city                |
+| Country Code        | B2C_CustomerAddress__x | CountryCode__c             | (None)                      | CustomerAddress | country_code        |
+| First Name          | B2C_CustomerAddress__x | FirstName__c               | (None)                      | CustomerAddress | first_name          |
+| Last Name           | B2C_CustomerAddress__x | LastName__c                | (None)                      | CustomerAddress | last_name           |
+| Phone               | B2C_CustomerAddress__x | Phone__c                   | (None)                      | CustomerAddress | phone               |
+| Postal Code         | B2C_CustomerAddress__x | PostalCode__c              | (None)                      | CustomerAddress | postal_code         |
+| State Code          | B2C_CustomerAddress__x | StateCode__c               | (None)                      | CustomerAddress | state_code          |
+| Customer List Id    | B2C_CustomerList__c    | Name                       | (None)                      | CustomerList    | id                  |
```

Comment:
Is this correct as Name?

Reply:
I'm not super familar with SFDC yet so I'm not sure what the Core ID and alt ID are for. Is one when you have a PersonAccount vs having a normal Contact and Account record?

Reply:
`Name` is correct, it is the standard name field of the object. All Core objects have one.

Regarding the Core ID vs Alt ID, as b2c-crm-sync is supporting both Account/Contact and PersonAccount models, we have to match the two cases.
For this, the Core ID is used to map fields with the Account/Contact model, where the Alt ID is used to map fields with PersonAccount model, in case the field is different than the Core ID one (for standard objects basically)